pos-print/src/main/java/com/clouway/pos/print/core/PrintNotifier.kt

```diff
@@ -0,0 +1,16 @@
+package com.clouway.pos.print.core
+
+/**
+ * Provides the methods to give feedback after a print
+ * has been attempted.
+ *
+ * @author Tsvetozar Bonev (tsvetozar.bonev@clouway.com)
+ */
+interface PrintNotifier {
```

Comment:
PrintNotifier doesn't sound good as interface. PrintingListener sounds better as name.

```kotlin
onReceiptPrinted(receipt: Receipt, printStatus: PrintStatus)
```

Reply:
Changed interface name to PrintingListener and implemented the suggested method signature.